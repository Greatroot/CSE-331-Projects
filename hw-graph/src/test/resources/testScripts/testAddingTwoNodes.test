#Test adding two nodes to graph | should add the two nodes to graph.

CreateGraph graph
AddNode graph n1
AddNode graph n2