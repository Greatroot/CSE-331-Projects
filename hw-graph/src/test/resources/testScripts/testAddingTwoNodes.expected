#Test adding two nodes to graph | should add the two nodes to graph.

created graph graph
added node n1 to graph
added node n2 to graph